jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -142,6 +142,8 @@ echo "Cloning pipeline repository $PIPELINE_GIT_URL"
 with_termination_log git clone "$PIPELINE_GIT_URL" . || exit 1
 echo "Checking out pipeline from revision $PIPELINE_GIT_REVISION"
 with_termination_log git checkout "$PIPELINE_GIT_REVISION" || exit 1
+echo "Delete pipeline git clone credentials"
+with_termination_log rm "~/.git-credentials" || exit 1
```

Comment:
Won't work because `~` is quoted and will not be expanded.

Fixed:
```suggestion
with_termination_log rm ~/.git-credentials || exit 1
```